### ISSUE TITLE
Clean Nehalem/Westmere ME 6.0

### ISSUE THREAD

**Issue by ValoWaking** [corna/me_cleaner#326]
Hi. I have very important question about Nehalem/Westmere Intel ME 6.0. Can me_cleaner clean all partitions (delete all firmware) on Intel ME 6.0 or not? 

In patch note 1.2v we have:

Wipe the ME6 Ignition firmware images
Fix the removal of the last partition on ME6

**Comment by Stitch626** [comment]
You can't "delete all firmware" cuz some regions are required by the chipset itself. The last one with fully removable (in this case replacable ;) ) firmware was the one used on the 775 non-Q chipsets.
So the short answer is NO.

**Comment by ValoWaking** [comment]
> 
> 
> You can't "delete all firmware" cuz some regions are required by the chipset itself. The last one with fully removable (in this case replacable ;) ) firmware was the one used on the 775 non-Q chipsets.
> So the short answer is NO.

No it's wrong. Try to download some 1156 bios and wipe it by your self. It's works and fully wipe.

**Comment by Stitch626** [comment]
I don't believe that. And even if it does, you may have the 30 minutes problem.

EDIT: After reading some similar things in another thread: be happy that it works.

**Comment by ValoWaking** [comment]
> 
> 
> I don't believe that. And even if it does, you may have the 30 minutes problem.

WatchDog can be fixed by Flash Image Tool and MMTool
